docs/HOT-FIX-README.md

```diff
@@ -0,0 +1,69 @@
+# Azure ACI plugin for Virtual Kubelet Downgrade setup
+
+## Installation
+
+Instructions for setup a downgrade for the previous official release using Helm.
+
+### Prerequisites
+
+- [Helm](https://helm.sh/docs/intro/quickstart/#install-helm)
+
+### Installing the chart
+
+1. Clone project
+```shell
+$ git clone https://github.com/virtual-kubelet/azure-aci.git
+$ cd helm
+```
+2. Install chart using Helm v3.0+
+```shell
+export CHART_NAME=virtual-kubelet-azure-aci-downgrade
+export VK_RELEASE=virtual-kubelet-azure-aci-1.4.1
+export NODE_NAME=virtual-kubelet-aci-1.4.1
+export CHART_URL=https://github.com/virtual-kubelet/azure-aci/raw/gh-pages/charts/$VK_RELEASE.tgz
+export MASTER_URI=$(kubectl cluster-info | awk '/Kubernetes control plane/{print $7}' | sed "s,\x1B\[[0-9;]*[a-zA-Z],,g")
+export IMG_URL=mcr.microsoft.com
+export IMG_REPO=oss/virtual-kubelet/virtual-kubelet
+export IMG_TAG=1.4.1
+export ENABLE_VNET=true
+
+# The following variables needs to be set according to the customer cluster.
```

Comment:
Add the following after line 30
(The information can be found in the Azure portal: AKS overview->Properties->Networking panel)

Reply:
done.